### ISSUE TITLE
Pressing the shortcut key should perform the associated action when the menu is open

### ISSUE THREAD

**Issue by ed-bassett** [jesseduffield/lazydocker#57]
**Is your feature request related to a problem? Please describe.**
A good way to learn the available keyboard shortcuts in any given context is to open the menu and see what's available. It would be good if I didn't have to manually navigate the menu to perform an action.

**Describe the solution you'd like**
I'd like the shortcut keys shown in a menu to perform the associated action

**Comment by jesseduffield** [comment]
I completely agree. @ed-bassett you truly have a strong grasp on good UX. If only we worked at the same company and I could regularly defer to your judgement about things like this.

I'll have a look into how straightforward something like this is (I suspect it's quite straightforward now that I've got a concept of a ParentView) and if so I might chuck a 'good first issue' label on this ticket. If you're in the mood for writing Go you might want to jump on this yourself ;)

**Comment by rgwood** [comment]
This is a really good idea, and it's tripped me up in both lazydocker and lazygit. In both cases (many months apart), I opened the menu by pressing x and couldn't figure out why the shortcuts shown in the menu didn't work.

(lazydocker and lazygit are both amazing, by the way. Thank you Jesse!)

**Comment by adamal** [comment]
To me, the current behaviour is so surprising I'd consider it a bug.

**Comment by jesseduffield** [comment]
This has been implemented in lazygit (see [here](https://github.com/jesseduffield/lazygit/blob/master/pkg/gui/context/menu_context.go#L104)) but not lazydocker. Unfortunately lazydocker is behind on some refactorings but it should be fairly straightforward to go through each menu item and add a keybinding for the key [here](https://github.com/jesseduffield/lazydocker/blob/master/pkg/gui/menu_panel.go#L102)
